src/components/data/SavedColumnSettings.js

```diff
@@ -213,6 +216,8 @@ const SavedColumnSettings = ({ workspaceId, snapshotName, entityType, entityMeta
       }, selectedSettingsNameExists ? 'Update' : 'Save')
     ] : [
       h(ButtonOutline, {
+        disabled: Boolean(editWorkspaceErrorMessage),
```

Comment:
You should assume disabled until you hear back from WorkspaceDetails. Otherwise there is a race condition during which the user could press the button before we AJAX call returns.